logs-to-tsv/query-logs.js

```diff
@@ -0,0 +1,69 @@
+const { CloudWatchLogsClient, StartQueryCommand, GetQueryResultsCommand } = require("@aws-sdk/client-cloudwatch-logs")
+const { writeFileSync } = require('fs')
+
+const client = new CloudWatchLogsClient({ accessKeyId: 'access key id', secretAccessKey: 'secret access key', region: 'us-east-1' });
```

Comment:
See [this](https://github.com/NYPL/discovery-hybrid-indexer/blob/b54478a4d8d81f5d56a91159ca551ef83906f923/lib/script-utils.js#L5-L14) as an alternative to handling creds manually. If you were to copy-paste that function into this script, you should be able to invoke the script like `node query-logs --profile nypl-digital-dev` and then within the script you'd just need to call `awsInit()` before instantiating your `CloudWatchLogsClient()`